makefile

```diff
@@ -43,3 +53,9 @@ clean:
 	cd fpga && rm -rf build
 	cd kicker && rm -rf build
 	rm -rf generated-docs
+
+CLANG_FORMAT_BINARY=clang-format-10
+
+pretty-lines:
+	@git diff -U0 --no-color $(DIFFBASE) | python3 util/style/clang-format-diff.py -binary $(CLANG_FORMAT_BINARY) -i -p1
+	@git diff -U0 --no-color $(DIFFBASE) | black .
```

Comment:
This was written with the expectation that the monorepo will be merged at some point soon. Until then, undo any changes done by make pretty-lines to the submodule (probably will break ci)

Reply:
but with monorepo this is no longer an issue. And it's not a big deal right now either, so I'd rather include the correct command in this pr.

Reply:
nvm I changed my mind about this. I'll update this line in the monorepo pr instead.